camel-core/src/test/java/org/apache/camel/language/simple/SimpleTest.java

```diff
@@ -463,14 +463,9 @@ public void testDateExpressions() throws Exception {
         cal.set(1974, Calendar.APRIL, 20);
         exchange.getIn().setHeader("birthday", cal.getTime());
 
+        assertExpression("date:header.birthday", cal.getTime());
         assertExpression("date:header.birthday:yyyyMMdd", "19740420");
-
-        try {
```

Comment:
As above, pattern part is now optional. I put the test back with the correct exception class error message.